images/kubevirt-infra-bootstrap/Dockerfile

```diff
@@ -16,3 +16,11 @@ RUN apt-get update \
         python3 \
         python3-pip \
         && rm -rf /var/lib/apt/lists/*
+RUN wget https://dl.google.com/go/go1.13.3.linux-amd64.tar.gz \
```

Comment:
Maybe it would be easier using gimme for the installation as done here https://github.com/kubevirt/cluster-network-addons-operator/blob/master/automation/check-patch.e2e.setup.sh? Not sure what is the better approach. Just for your consideration.

Reply:
I agree - gimme approach more elegant.